### PR TITLE
Event post for Lunch with Microsoft

### PR DESCRIPTION
**Pull request by shivangipatwardhan2018**
Please let me know if anything is to be changed.

### CODE REVIEW

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (arshiamufti) [inline content/F2016/F16-Lunch-with-Microsoft.md]: - software engineer, program manager, etc should be all lowercase
- this is a nitpicky thing but I'm not sure that values should be separated by semi-colons. "Ankita Dey (software engineer), Kyle Oline (program manager on Windows Internet of Things)...."

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (arshiamufti) [inline content/F2016/F16-Lunch-with-Microsoft.md]: include the date: Tuesday, September 27

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (arshiamufti) [inline content/F2016/F16-Lunch-with-Microsoft.md]: "current full time engineers" would be better

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (arshiamufti) [inline content/F2016/F16-Lunch-with-Microsoft.md]: `&ndash;` instead of the dash

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (arshiamufti) [inline content/F2016/F16-Lunch-with-Microsoft.md]: update: I was wrong about the semicolons. http://www.bristol.ac.uk/arts/exercises/grammar/grammar_tutorial/page_05.htm @shivangipatwardhan2018 pointed this out.

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (shivangipatwardhan2018) [reply content/F2016/F16-Lunch-with-Microsoft.md]: Yes you are right. Fixed it :)

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (arshiamufti) [reply content/F2016/F16-Lunch-with-Microsoft.md]: (I noticed the tea party event didn't have that but most of our past event posts do and it's clearer to include the date anyway)

**Comment by fboxwala** [comment]
Nice work - could you just squash your commits into one commit, and next time make sure you're working off of your own feature branch and not master?

(in case you need it -  [guide to squash](https://github.com/wics-uw/git-workshop-W16/wiki/Squashing-Commits))

**Comment by shivangipatwardhan2018** [comment]
Yes will do.

**Comment by arshiamufti** [comment]
So....this wasn't merged in before the event.

**Comment by evykassirer** [comment]
@shivangipatwardhan2018 can you make the changes people asked for so we can merge it?

**Comment by shivangipatwardhan2018** [comment]
I did a while back I am not aware there was anything else to change.

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (evykassirer) [inline content/F2016/F16-Lunch-with-Microsoft.md]: ndash here too?

**Review by evykassirer — CHANGES_REQUESTED** [review]
you included the review, but I don't see the ndash or the squashed commit. maybe you changed it but didn't push? sorry for the confusion!

**Comment by fboxwala** [comment]
@shivangipatwardhan2018 Oh sorry, I was waiting for you to squash the commits before I merged, and I guess we didn't touch base fast enough. Sorry about the confusion

**Comment by shivangipatwardhan2018** [comment]
okay I was not aware. I will do it after work today

**Comment by evykassirer** [comment]
one cool thing is that we can do squash and merge now when we merge pull requests

so that means you could actually just not squash them and I can do it for you when I merge

screen shot 2016-09-28 at 10 21 26 am

You're welcome to try squashing on your own anyways if you want, and I think adding the ndash would be good too.

**Comment by shivangipatwardhan2018** [comment]
Evy you were right the push had not gone though fixed the ndash issue also all should be okay now. :)

**Comment by evykassirer** [comment]
oh haha so you did it but just didn't push - classic 😄

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (evykassirer) [inline content/F2016/F16-Lunch-with-Microsoft.md]: oh one more thing - lines should only be 80 characters long
when you do a new line it won't actually do a line break on the website, it just makes the file cleaner and easier to read (this is common for coding too)

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (evykassirer) [inline content/F2016/F16-Lunch-with-Microsoft.md]: here too

**Review by evykassirer — CHANGES_REQUESTED** [review]

File: content/F2016/F16-Lunch-with-Microsoft.md
Comment (annalorimer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: I customized my text editor to newline at 80 characters and I haven't seen any issues?

Comment (evykassirer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: this isn't your PR though? am confuse
this line is 102 characters long and line 11 is longer

Comment (annalorimer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: Oh I meant that I'm confused about "when you do a new line it won't actually do a line break"

Comment (evykassirer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: oh, yeah I explained that poorly
I wanted to say that new lines in the code doesn't mean line breaks when it shows up on the website

Comment (annalorimer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: Can't @shivangipatwardhan2018 just newline after 80 chars to get the correct line lengths?

edit: you answered this at the same time i posted :)

Comment (evykassirer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: yeah exactly - there should be newline after every 80 chars

but that doesn't mean that on the website every line will cut off after 80 characters

Comment (annalorimer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: Ah ok we're saying the same thing. @shivangipatwardhan2018 does this make sense?

Comment (evykassirer) [reply content/F2016/F16-Lunch-with-Microsoft.md]: yeah I was just trying to give context hehe

**Comment by evykassirer** [comment]
👌 👌 👌 👌 👌 👌 👌 👌

**Review by evykassirer — APPROVED** [review]

**Pull request merged**
